src/plugins/intel_cpu/src/nodes/mkldnn_pad_node.cpp

```diff
@@ -331,6 +331,10 @@ void MKLDNNPadNode::executeDynamicImpl(mkldnn::stream strm) {
     execute(strm);
 }
 
+std::vector<VectorDims> MKLDNNPadNode::shapeInfer() const {
+    return MKLDNNNode::shapeInferGeneric(PortMask(PADS_BEGIN_ID, PADS_END_ID));
```

Comment:
One question that I have regarding current ShapeInference API design:
Currently a plugin should determine which data (from which ports) should be passed to shape infer on its own. In other words it utilizes some implicit knowledge that particular shape_inference requires data only from particular ports (Pad is an example: we don't need to pass pad_value). We still can pass all constans, but it will introduce additional overheads on host_tensor creation.
**So my question is: shouldn't shapeInfer provide API that returns port mask for requested constants?** 
@jane-intel  @usstq  @maxnick what do you think?